spec/api/location_types_spec.rb

```diff
@@ -5,7 +5,8 @@
 require "rails_helper"
 
 RSpec.describe Api::LocationTypesController, type: :request do
-  let!(:administrator) { create(:administrator) }
+  let!(:admin_swipe_card_id) { "SwipeCardId:10" }
```

Comment:
One of the things you can do is create a factory sequence to automate the creation of swipe_card_ids.